.github/workflows/flowzone.yml

```diff
@@ -28,6 +28,6 @@ jobs:
       runs_on: '["ubuntu-20.04"]'
       docker_runs_on: >
         {
-          "linux/amd64": ["self-hosted","distro:focal","X64"],
+          "linux/amd64": ["actuated-4cpu-16gb","focal"],
```

Comment:
Should we add the comment that we didn't change the arm runner just b/c the actuated ones only have focal for amd64 atm?